### PR TITLE
Feature/jats notebook

### PR DESCRIPTION
**Pull request by dragonstyle**

### CODE REVIEW

File: src/resources/filters/quarto-post/book.lua [hunk 1]
```diff
@@ -1,6 +1,9 @@
 -- book.lua
 -- Copyright (C) 2020-2022 Posit Software, PBC
 
+local authors = require '../common/authors'
+local license = require '../common/license'
+
```
Comment (cscheid) [inline src/resources/filters/quarto-post/book.lua:6]: 🤩

Comment (dragonstyle) [reply src/resources/filters/quarto-post/book.lua]: This has caused me to enter LUA inlining nightmare land. See next commit, then following failure:

```
too many local variables (limit is 200) in main function near '='
[326](https://github.com/quarto-dev/quarto-cli/actions/runs/4833870917/jobs/8614410733?pr=5348#step:5:327)
```

salute

**Review by cscheid — APPROVED** [review]
Beast PR, looks good!

(Only one minor question on moving the embed shortcode resolution from pre-engine to post engine. I'm sure there's a good reason for it, but I'm wondering if it'll break user filters that looked at the embed contents.)

File: src/resources/filters/quarto-post/book.lua (hunk 1 above)
Comment (cscheid) [reply src/resources/filters/quarto-post/book.lua:6]: That's not terrifying at all

**Pull request merged**